images/weaviate/configs/latest.apko.yaml

```diff
@@ -9,6 +9,11 @@ paths:
     permissions: 0o777
     uid: 65532
     gid: 65532
+  - path: /var/lib/weaviate
+    type: directory
+    permissions: 0o777
```

Comment:
not blocking, just curious why 777 is needed

Reply:
Fixed to 755

Reply:
why does it need to be 0755?

What qualifies fox for this PR?